script.js

```diff
@@ -10,38 +10,46 @@ const createCardList = (array) => {
   //   container.innerHTML = "";
 
   array = array.centers;
-  array.forEach((obj) => {
-    const card = document.createElement("div");
-    card.classList.add("myDiv");
-    let temp;
+  array.forEach((obj1) => {
```

Comment:
we could add a different variable name instead of obj1 for meeting code standards.